hack/fix_copyright_banner.sh

```diff
@@ -1,19 +1,5 @@
 #!/bin/bash
 
-# Copyright Istio Authors
```

Comment:
kind of remeber this is from istio repo. If so we should keep it

Reply:
reverted